OPAL/ba/src/test/scala/org/opalj/ba/ClassFileBuilderTest.scala

```diff
@@ -89,19 +89,46 @@ class ClassFileBuilderTest extends AnyFlatSpec {
             attributes = RefArray(permittedSubclassesAttribute, br.Synthetic)
         ).toDA()
 
+    val (nestedClassInner, _) =
+        CLASS[Nothing](
+            version = UShortPair(0, 49),
```

Comment:
Is this the proper Java version? Also, I suggest we move this upwards so features introduced later (i.e., with a higher version number here) are later in this file.